contrib/README.md

```diff
@@ -37,15 +37,34 @@ It will work in the directory you called it from. So if you wish to use it, I'd
 recommend you either put it into your `$PATH` or create yourself an alias for it.
 
 Once you've set it up and have it running, you could even have a Cron job
-taken care for regular runs. Here I'd recommend the following command line:
+taken care for regular runs. With the config set up properly, the command
+for that job could look like this:
 
 ```bash
-cd /path/to/archive && mastoarch -a Me@MyInstance 2>/dev/null
+source /home/johndoe && mastoarch archive
 ```
 
-If you've initialized a git repo with your archive, you could include `-g 1`
+or, alternatively:
+
+```bash
+cd /path/to/archive && mastoarch -a Me@MyInstance -b "" 2>/dev/null
+```
+
+If you've initialized a git repo with your archive, you could include `-g 2`
 with the call to have changes committed automatically.
 
+## `mastosearch` (by Izzy)
+For your impromptu searches. Again a wrapper around the main script intended
+to be easy to use. With your defaults set up in the configuration, a search
+is as easy as `mastosearch <searchterm>`. Again, the full syntax is shown when
+called without parameters.
+
+## `config.sample` (by Izzy)
+Lets you define defaults to use with `mastoarch` and `mastosearch` to make
+calls to them as simple as possible. On your Linux machine, copy it to
+`${HOME}/.config/mastodon-backup/config` and adjust it to fit your needs.
```

Comment:
mastodon-archive, as discussed below

Reply:
Another option would be to check if the config file exists and if it does not, querying the user for values.

Reply:
The latter is done. Default values are defined in the script, then the config is loaded if it exists, then command-line parameters are processed. The only interactive question is whether to commit to git, and that only if `autobackup=1` (not with `0` or `2`). Adding a hint at this place.

Reply:
The former is done as well. If you are OK with how I handle defaults (avoiding interactively prompting the users), this conversation could then be resolved.